src/databricks/labs/ucx/framework/logger.py

```diff
@@ -1,6 +1,8 @@
 import logging
 import sys
 
+logger = logging.getLogger(__name__)
```

Comment:
Why is it here?

Reply:
This is a leftover from earlier, removed.